### PR TITLE
patch for net-data script

### PR DESCRIPTION
**Pull request by hathyaga**
Patch fixes testinterface execution for interface check based
on timeout

Signed-off-by: Harsha Thyagaraja <harshkid@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by hathyaga** [comment]
Job log
[net_data_joblog.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/902320/net_data_joblog.txt)

File: io/net/net_data.py [hunk 1]
```diff
@@ -158,12 +158,14 @@ def testinterface(self):
...
-        ret = process.system_output(self.eth, shell=True)
-        if 'no' in ret:
+        for i in range(0, 600, 60):
+            if 'yes' in process.system_output(self.eth, shell=True):
+                break
+            time.sleep(60)
+        if 'no' in process.system_output(self.eth, shell=True):
             self.fail("interface test failed")
-        # check the status of interface through ip link show
+            # check the status of interface through ip link show
```
Comment (narasimhan-v) [inline io/net/net_data.py]: why indentation for comment ?

File: io/net/net_data.py [hunk 2]
```diff
@@ -158,12 +158,14 @@ def testinterface(self):
             self.fail("interface test failed")
         # up the interface
         process.system(if_up, shell=True)
-        time.sleep(4)
         # check the status of interface through ethtool
-        ret = process.system_output(self.eth, shell=True)
-        if 'no' in ret:
+        for i in range(0, 600, 60):
```
Comment (narasimhan-v) [inline io/net/net_data.py]: A comment can explain what you are doing here better

**Comment by PraveenPenguin** [comment]
@narasimhan-v thanks

**Pull request merged**